R/cvcov.R

```diff
@@ -0,0 +1,16 @@
+#' Clustered variance-covariance matrices and T statistic d.o.f.
```

Comment:
I think so; it's used in an example and someone might be like "what the heck is this"